src/server/debug/server/server.go

```diff
@@ -927,3 +955,150 @@ func redirectDump(ctx context.Context, c debug.DebugClient, filter *debug.Filter
 	}
 	return grpcutil.NewStreamingBytesReader(dumpC, nil), nil
 }
+
+func (s *debugServer) collectDatabaseStats(ctx context.Context, tw *tar.Writer, prefix ...string) error {
+	if err := s.collectDatabaseActivities(ctx, tw, prefix...); err != nil {
+		return err
+	}
+	if err := s.collectDatabaseRowCount(ctx, tw, prefix...); err != nil {
+		return err
+	}
+	if err := s.collectDatabaseTableSizes(ctx, tw, prefix...); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (s *debugServer) collectDatabaseActivities(ctx context.Context, tw *tar.Writer, prefix ...string) error {
+	return s.collectDatabaseQuery(ctx, tw, "activities", queryDatabaseActivities, prefix...)
+}
+
+func (s *debugServer) collectDatabaseRowCount(ctx context.Context, tw *tar.Writer, prefix ...string) error {
+	return s.collectDatabaseQuery(ctx, tw, "row-counts", queryDatabaseRowCounts, prefix...)
+}
+
+func (s *debugServer) collectDatabaseTableSizes(ctx context.Context, tw *tar.Writer, prefix ...string) error {
+	return s.collectDatabaseQuery(ctx, tw, "table-sizes", queryDatabaseTableSizes, prefix...)
+}
+
+func (s *debugServer) collectDatabaseQuery(ctx context.Context, tw *tar.Writer, fileName string, queryFunc func(conn *sql.Conn, w io.Writer) error, prefix ...string) error {
+	return collectDebugFile(tw, fileName, "txt", func(w io.Writer) error {
+		conn, err := s.env.GetDBClient().Conn(ctx)
+		if err != nil {
+			return errors.EnsureStack(err)
+		}
+		defer conn.Close()
+
+		err = queryFunc(conn, w)
+		if err != nil {
+			return errors.EnsureStack(err)
+		}
+		return errors.EnsureStack(err)
+	}, prefix...)
+}
+
+func queryDatabaseRowCounts(conn *sql.Conn, w io.Writer) error {
+	query := `
+		SELECT schemaname,relname,n_live_tup 
```

Comment:
I think we might as well pull some more columns here:
`seq_scan`, `idx_scan` both look helpful